data/food/base/data.json

```diff
@@ -68,6 +68,41 @@
             "fdc_id": "168936",
             "alias": [],
             "comment": "all purpose flour, equivalent to type 550"
+        },
+        "food-ketchup": {
+            "name": "Ketchup",
+            "plural_name": "Ketchup",
+            "supermarket_category": "category-baking",
+            "preferred_unit_metric": "unit-g",
+            "preferred_packaging_unit_metric": "unit-g",
```

Comment:
My recipes all use ml and not grams for ketchup. Interestingly, mayo is measured in grams. The ketchup in my fridge is labeled in ml. The mayo in ml as well as g.

Reply:
ml updated. Thanks!